From 2a54c8b1663321ecd7b431ad0d9922834a3477bd Mon Sep 17 00:00:00 2001
From: Kevin <kevinpacaud@gmail.com>
Date: Fri, 6 Mar 2026 00:14:42 +0000
Subject: [PATCH] Ignore CDN build output

---
 .gitignore | 1 +
 1 file changed, 1 insertion(+)

diff --git a/.gitignore b/.gitignore
index e4953be..808b1ba 100644
--- a/.gitignore
+++ b/.gitignore
@@ -37,3 +37,4 @@ scripts/sp_deploy.env
 scripts/*.env.local

 framework/cdn/
+cdn/
--
2.47.3